elastic/fleet-server
# [Fleet] Fake agent in fleet server standalone

## Description 

Resolve #2605

This create a fake agent id and version for fleet server standalone to allow the coordinator to work.

## How to test?

Run a standalone Fleet server and verify that you can enroll an agent and the agent is fully functional (get the policy ...)
